tests/e2e/tests/admin/transfer-tokens.spec.ts

```diff
@@ -26,12 +26,19 @@ const createTransferToken = async (page, tokenName, duration, type) => {
 };
 
 test.describe('Transfer Tokens', () => {
-  test.beforeEach(async ({ page }) => {
+  let page: Page;
```

Comment:
Is this a pattern we want to promote? you effectively make the last test dependent on the previous because you're not starting with a "fresh" slate every time. 

I'd be cautious about promoting this pattern as I don't think it's something we want to get into the habit of doing in the long term?

Reply:
> Since we aren't clearing the db after each run anymore and have multiple tokens, we need to be more specific with what we're looking for

My point exactly.

Reply:
Unfortunately I think the bad practice is the core of the fix here. I will take a look and see if there's an alternative I can find, but otherwise I may just add a warning not to repeat that pattern in other tests.